package.json

```diff
@@ -1,39 +1,35 @@
 {
   "name": "kilt-extension-api",
   "version": "0.1.0",
-  "main": "./index.js",
-  "types": "./index.d.ts",
-  "type": "commonjs",
+  "main": "./dist/cjs/index.js",
+  "types": "./dist/cjs/index.d.ts",
+  "type": "module",
   "exports": {
-    ".": {
-      "types": "./index.d.ts",
-      "default": "./index.js"
-    },
-    "./wellKnownDidConfiguration": {
-      "types": "./wellKnownDidConfiguration/index.d.ts",
-      "default": "./wellKnownDidConfiguration/index.js"
-    },
-    "./types": {
-      "types": "./types/index.d.ts"
-    }
+    "import": "./dist/esm/index.js",
+    "require": "./dist/cjs/index.js",
+    "default": "./dist/cjs/index.js"
   },
   "author": "KILT <info@kilt.io>",
   "license": "BSD-4-Clause",
   "scripts": {
     "lint": "eslint --ext .ts .",
-    "build": "tsc -p tsconfig.build.json",
+    "build": "yarn build:cjs && yarn build:esm",
+    "build:cjs": "tsc -p tsconfig.cjs.json",
+    "build:esm": "tsc -p tsconfig.json",
     "clean": "exec git clean -xf -e 'node_modules' '*'",
```

Comment:
I don't think these changes are intended?

Reply:
Yes, sorry. Yesterday we merged the cjs esm setup into main. Therefore, the whole build scripts were wrong :smile: I merged main again. The package.json and tsconfigs should be fine now